### ISSUE TITLE
Add a "paste from clipboard" button to adress fields

### ISSUE THREAD

**Issue by IfaqYurmama** [NAVCoin/NavPay#52]
Would make mobile sending even easier

**Comment by craigmacgregor** [comment]
you can just long press and use the default paste functionality though right?

We could look to add something like this after public beta is launched.

**Comment by IfaqYurmama** [comment]
Yes you can but for me I have to access the field, click in the field again to bring up the arrow, click on the arrow and then select paste. At least for me it would be more comfortable to have a paste button in the UI.
